tests/DCPS/Serializer/SerializerTest.cpp

```diff
@@ -695,6 +696,41 @@ bool runOverrunTest()
   return true;
 }
 
+bool runEncapsulationOptionsTest()
+{
+  std::cerr << "\nRunning encapsulation options tests...\n";
+  const CORBA::Octet arr[4] = {0};
+  const OpenDDS::DCPS::EncapsulationHeader encap;
+  OpenDDS::DCPS::unique_ptr<OpenDDS::DCPS::MessageBlockAllocator> mb_allocator_;
```

Comment:
```suggestion```

Reply:
I don't know why it showed up like this, but this isn't being used and the suggestion is to remove the line.